hack/snyk-report.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+cat > docs/snyk/index.md <<- EOM
+# Snyk Scans
+
+Every Sunday, Snyk scans are generated for Argo CD's \`master\` branch and the most recent patches of the three most
+recent minor releases.
+
+The [latest](https://argo-cd.readthedocs.io/en/latest/snyk/) version of this page shows the most recent scan.
+The [stable](https://argo-cd.readthedocs.io/en/stable/snyk/) version of this page shows the last scan before the most
+recent stable release was cut. The latest version will have the most up-to-date information.
+
+## Scans
+EOM
+
```

Comment:
Want to do all that follows in a temporary directory created by `mktemp` maybe and then move files to the location of the real repository?